data/explorations/all_interactions/all_interactions.yaml

```diff
@@ -214,6 +214,8 @@ states:
           value: true
         canAddVertex:
           value: true
+        interactionIsActive:
```

Comment:
No, this shouldn't be part of the YAML. The schema of the exploration should not be changed, since this would break all the existing explorations in our datastore in production. If there are race issues, we'll need to find a different way to deal with them.

Ditto below.

Reply:
Done, removed those.